headers/functions/arm9.h

```diff
@@ -196,6 +196,7 @@ enum move_id GetItemMoveId(enum item_id item_id);
 bool TestItemAiFlag(enum item_id item_id, enum item_flag flag);
 bool IsItemInTimeDarkness(enum item_id item_id);
 bool IsItemValidVeneer(enum item_id item_id);
+enum item_id ReturnEggExclusiveItem(enum monster_id);
```

Comment:
Still give the param a name though, e.g.

```suggestion
enum item_id ReturnEggExclusiveItem(enum monster_id monster_id);
```